packages/search-ui-elasticsearch-connector/src/handlers/search/Configuration.ts

```diff
@@ -18,13 +18,7 @@ export function getResultFields(
   hitFields: string[];
   highlightFields: string[];
 } {
-  const hitFields = Object.keys(resultFields).reduce((sum, fieldKey) => {
-    const fieldConfiguration = resultFields[fieldKey];
-    if (fieldConfiguration.raw) {
-      sum.push(fieldKey);
-    }
-    return sum;
-  }, []);
+  const hitFields = Object.keys(resultFields);
```

Comment:
this is unrelated to the bug but required from the bump of searchkit. Found an issue where we need all fields (raw and highlighted) to be hitFields.